### DISCUSSION TITLE
Log alerts to json line delimited file

### DISCUSSION THREAD

**Discussion by theflakes** [jertel/elastalert2#1102] [Q&A] [answered]
I'm new to Elastalert2. Trying to figure out how to write every alert to a line delimited json file. I'm sure I have to do this with the "command" rule directive, but unable to figure out exactly what I need to do to include alert_text, rule name and other information in the json alert log.

thanks all

**Comment by jertel** [comment, marked as answer]
You can use the `debug` alerter to send all alerts to a log file. You can configure the logger format via the `logging:` section of `config.yaml`.

If you prefer to use the `command` alerter, that will work also.

As there are examples and documentation already provided for ElastAlert 2, we would appreciate it if you are more specific as to what you've tried, the expected outcome vs what actually occurred. See #11 for more information about requesting community support.
